src/routes/about/+page.svelte

```diff
@@ -115,15 +115,10 @@
   <h2>Meet The Team</h2>
 
   <Testimonial
```

Comment:
Would this be a good place to use the testimonial carousel?

Reply:
I think so, I'll do it in this PR.